iree/compiler/Dialect/HAL/Target/LLVM/LLVMAOTTarget.cpp

```diff
@@ -65,15 +65,27 @@ class LLVMAOTTargetBackend final : public TargetBackend {
   explicit LLVMAOTTargetBackend(LLVMTargetOptions options)
       : options_(std::move(options)) {}
 
-  // NOTE: we could vary these based on the options, such as by arch/etc.
   std::string name() const override { return "llvm"; }
 
-  std::string deviceID() const override { return "dylib"; }
+  std::string deviceID() const override { return "cpu"; }
```

Comment:
agreed! the goal for the runtime is that you'll only specify the device and not the codegen backend - so 'cpu' (or whatever) instead of 'dylib' - as the local CPU backend can handle anything it can load via a registered loader.